concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+ID	NUMBER	SPANISH	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Lastra-1986-381-1	1	agua	water	948	WATER
+Lastra-1986-381-2	2	fuego	fire	221	FIRE
+Lastra-1986-381-3	3	humo	smoke	778	SMOKE (EXHAUST)
+Lastra-1986-381-4	4	hollín	soot		
+Lastra-1986-381-5	5	ceniza	ash	646	ASH
+Lastra-1986-381-6	6	carbón	coal	1188	CHARCOAL
+Lastra-1986-381-7	7	tierra	land	1228	EARTH (SOIL)
+Lastra-1986-381-8	8	arena	sand	671	SAND
+Lastra-1986-381-9	9	cerro	hill	639	MOUNTAIN
+Lastra-1986-381-10	10	cerros	hills		
+Lastra-1986-381-11	11	río	river	666	RIVER
+Lastra-1986-381-12	12	lama (en el agua)	Lama (in water)		
+Lastra-1986-381-13	13	cielo	sky	1732	SKY
+Lastra-1986-381-14	14	luna (en caso de préstamo, preguntar "mes")	moon	1313	MOON
+Lastra-1986-381-15	15	sol	sun	1343	SUN
+Lastra-1986-381-16	16	estrella	star	1430	STAR
+Lastra-1986-381-17	17	luz	light	630	LIGHT (RADIATION)
+Lastra-1986-381-18	18	sombra (de un árbol)	shadow (of a tree)	1388	SHADE
+Lastra-1986-381-19	19	viento	wind	960	WIND
+Lastra-1986-381-20	20	lluvia	rain	658	RAIN (PRECIPITATION)
+Lastra-1986-381-21	21	llovizna	drizzle	1016	DRIZZLE
+Lastra-1986-381-22	22	relámpago	flash of lightning	1654	LIGHTNING
+Lastra-1986-381-23	23	trueno	thunder	1150	THUNDER
+Lastra-1986-381-24	24	nube	cloud	1489	CLOUD
+Lastra-1986-381-25	25	neblina	fog	249	FOG
+Lastra-1986-381-26	26	cabeza (sin posesivo)	Head (without possessive)	1256	HEAD
+Lastra-1986-381-27	27	cabello (sin posesivo)	hair (without possessive)	2648	HAIR (HEAD)
+Lastra-1986-381-28	28	cara	face	1560	FACE
+Lastra-1986-381-29	29	frente	forehead	123	FOREHEAD
+Lastra-1986-381-30	30	mi ojo	my eye	1248	EYE
```

Comment:
Below some concepts with possessives are unmapped (i.e. my nail), here they are mapped. I'd opt for all possessive versions to be unmapped.

Reply:
Oh yes, those mappings escaped my sight. Thanks for pointing this out. I'd also unmap the possessive versions. Specially 'cause there are already unpossessed forms for the same lexemes on some cases.

Reply:
Sounds good, thanks for checking!